directives/annotated.py

```diff
@@ -357,11 +366,42 @@ def annotate(html, section_name, annotations):
     content = u''.join(result)
     return u''.join([parts[0], parts[1], content, parts[3], parts[4]])
 
-
+def add_assets(app):
+    # This method reads the `include_annotated_css` and `include_annotated_js`
+    # settings from the conf.py file located in the course directory. If such
+    # settings are not found, the default settings defined in the setup()
+    #  method will be used instead
+    app.config.include_annotated_css and app.add_stylesheet(CSS_FILE)
```

Comment:
You are using the old config variable, but you switched to theme options. In addition, `app.add_stylesheet` is not needed with the theme options. The same for JS.